flow/connectors/utils/aws.go

```diff
@@ -72,6 +80,22 @@ func GetAWSSecrets(creds S3PeerCredentials) (*AWSSecrets, error) {
 	}, nil
 }
 
+func GetClickhouseAWSSecrets(bucketPathSuffix string) *ClickhouseS3Credentials {
+	awsRegion := os.Getenv("PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_REGION")
+	awsKey := os.Getenv("PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_ACCESS_KEY_ID")
+	awsSecret := os.Getenv("PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_SECRET_ACCESS_KEY")
+	awsBucketName := os.Getenv("PEERDB_CLICKHOUSE_AWS_S3_BUCKET_NAME")
+
+	escapedPathSuffix := url.PathEscape(bucketPathSuffix)
+	awsBucketPath := fmt.Sprintf("s3://%s/%s", awsBucketName, escapedPathSuffix)
```

Comment:
```suggestion
	awsBucketPath := fmt.Sprintf("s3://%s/%s", awsBucketName, bucketPathSuffix)
```
Reading back, we have this path being `%s/%s` so seems we don't want to escape it here, escaping user input needs to happen higher up the call chain